core/src/Streamly/Data/Unfold.hs

```diff
@@ -85,6 +85,9 @@ module Streamly.Data.Unfold
     -- ** Nesting
     , many
 
+    -- ** Enumerating 'Num' Types
+    , enumerate
+    , enumerateTo
```

Comment:
We have to expose the Enumerable type class and its members as well, that was the main thing to do in this PR.